### PR TITLE
Adding documentation for HTTP request in Browser tests

### PR DESCRIPTION
**Pull request by MargotLepizzera**
### What does this PR do?

This PR adds documentation for HTTP requests in Browser tests
This mostly duplicates already existing content from https://docs.datadoghq.com/synthetics/api_tests/?tab=httptest#make-a-request

### Preview link

https://docs-staging.datadoghq.com/margot.lepizzera/http_requests_browser_tests/synthetics/browser_tests/actions/#http-requests

### CODE REVIEW

File: content/en/synthetics/browser_tests/actions.md [hunk 1]
```diff
@@ -164,6 +164,45 @@ You can run browser tests within other browser tests, up to two levels of nestin
 
 **Note**: If it does not make sense for you to run your subtest independently, you can pause it. It will continue to be called as part of your main test, but it will not be executed individually.
 
+## HTTP Requests
```
Comment (ruthnaebeck) [inline content/en/synthetics/browser_tests/actions.md:167]:
```suggestion
## HTTP requests
```

File: content/en/synthetics/browser_tests/actions.md [hunk 2]
```diff
@@ -164,6 +164,45 @@ You can run browser tests within other browser tests, up to two levels of nestin
...
+
+You can run HTTP requests as part of your Browser tests.
```
Comment (ruthnaebeck) [inline content/en/synthetics/browser_tests/actions.md:169]:
```suggestion
You can run HTTP requests as part of your browser tests. 
```

File: content/en/synthetics/browser_tests/actions.md [hunk 3]
```diff
@@ -164,6 +164,45 @@ You can run browser tests within other browser tests, up to two levels of nestin
 
 **Note**: If it does not make sense for you to run your subtest independently, you can pause it. It will continue to be called as part of your main test, but it will not be executed individually.
 
+## HTTP Requests
+
+You can run HTTP requests as part of your Browser tests. 
+
+{{< img src="synthetics/browser_tests/recorder_http_requests.png" alt="HTTP Request step"  style="width:70%;" >}}
+
+### Perform an HTTP request
```
Comment (ruthnaebeck) [inline content/en/synthetics/browser_tests/actions.md:173]:
```suggestion
### Setup
```

File: content/en/synthetics/browser_tests/actions.md [hunk 4]
```diff
@@ -164,6 +164,45 @@ You can run browser tests within other browser tests, up to two levels of nestin
...
+
+To define your HTTP request:
+
+1. Choose the **Method** and **URL** to query. Available methods are: `GET`, `POST`, `PATCH`, `PUT`, `HEAD`, `DELETE`, and `OPTIONS`.
+2. Optionally specify **Advanced Options**:
+     * Follow redirects: Toggle to have the monitored endpoint follow up to ten redirects.
+     * Allow insecure certificates: Toggle to have your HTTP test go on with connection even if there is an error when validating the certificate.
```
Comment (ruthnaebeck) [inline content/en/synthetics/browser_tests/actions.md:180]:
```suggestion
     * Allow insecure certificates: Toggle to have your HTTP test continue the connection even if there is an error when validating the certificate.
```

File: content/en/synthetics/browser_tests/actions.md [hunk 5]
```diff
@@ -164,6 +164,45 @@ You can run browser tests within other browser tests, up to two levels of nestin
...
+     * Headers: Defined headers override the default browser headers. For example, set the User Agent in the header to [identify Datadog scripts][1].
```
Comment (ruthnaebeck) [inline content/en/synthetics/browser_tests/actions.md:181]:
```suggestion
     * Headers: Defined headers override the default browser headers, for example: set the User Agent in the header to [identify Datadog scripts][1].
```

File: content/en/synthetics/browser_tests/actions.md [hunk 6]
```diff
@@ -164,6 +164,45 @@ You can run browser tests within other browser tests, up to two levels of nestin
...
+     * Authentication: HTTP basic authentication with username and password
+     * Body: Request body and body type (`text/plain`, `application/json`, `text/xml`, `text/html`, or `None`). **Note**: The request body is limited to a maximum size of 50 kilobytes.
+     * Cookies: Defined cookies are added to the default browser cookies. Set multiple cookies using the format `<COOKIE_NAME1>=<COOKIE_VALUE1>; <COOKIE_NAME2>=<COOKIE_VALUE2>`.
+3. Click on **Test URL** to try out your request configuration. You should see a response preview show up allowing you to confirm whether you are getting the expected response data.
```
Comment (ruthnaebeck) [inline content/en/synthetics/browser_tests/actions.md:185]:
```suggestion
3. Click **Test URL** to test your request configuration. This results in a preview showing response data.
```

File: content/en/synthetics/browser_tests/actions.md [hunk 7]
```diff
@@ -164,6 +164,45 @@ You can run browser tests within other browser tests, up to two levels of nestin
...
+
+{{< img src="synthetics/browser_tests/http_request.png" alt="Make HTTP Request"  style="width:80%;" >}}
+
+### Add assertions
+
+You can also optionally decide to base your step success on assertions about the defined HTTP request:
```
Comment (ruthnaebeck) [inline content/en/synthetics/browser_tests/actions.md:191]:
```suggestion
Optionally, you can base your step success on assertions about the defined HTTP request:
```

File: content/en/synthetics/browser_tests/actions.md [hunk 8]
```diff
@@ -164,6 +164,45 @@ You can run browser tests within other browser tests, up to two levels of nestin
...
+
+| Type          | Operator                                                                        | Value type                 |
+|---------------|---------------------------------------------------------------------------------|----------------------------|
+| Status Code   | `is`, `is not`                                                                  | _Integer_                  |
+| Response time | `lessThan`                                                                      | _Integer (ms)_             |
+| Headers       | `contains`, `does not contain`, `is`, `is not` <br> `matches`, `does not match` | _String_ <br> _[Regex][7]_ |
+| Body          | `contains`, `does not contain`, `is`, `is not` <br> `matches`, `does not match` | _String_ <br> _[Regex][7]_ |
+
```
Comment (ruthnaebeck) [inline content/en/synthetics/browser_tests/actions.md:199]: I think this table would be easier to read if the `Value type` column was in the center.

File: content/en/synthetics/browser_tests/actions.md (hunk 5 above)
Comment (MargotLepizzera) [reply content/en/synthetics/browser_tests/actions.md]: Removed the ref to user agent as a user agent is set by default by our tests

File: content/en/synthetics/browser_tests/actions.md (hunk 8 above)
Comment (MargotLepizzera) [reply content/en/synthetics/browser_tests/actions.md:199]: Done!

**Pull request merged**